packages/p2p/src/pages/advertiser-page/advertiser-page-adverts/advertiser-page-adverts-table-header.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { DesktopWrapper, Table } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from 'Components/i18next';
+
+const AdvertiserPageAdvertsTableHeader = () => {
+    const {
+        client: { currency },
+    } = useStore();
+
+    return (
+        <DesktopWrapper>
+            <Table.Header>
+                <Table.Row className='advertiser-page-adverts__table-row'>
```

Comment:
```suggestion
                <Table.Row className='advertiser-page-adverts-row'>
```

isnt it supposed to be like this? since u rename `__table-row` to `-row` in `advertiser-page-adverts.scss`

Reply:
no. i had added -row under __table.